src/index.js

```diff
@@ -0,0 +1,149 @@
+import './style.css';
+import '../node_modules/@fortawesome/fontawesome-free/js/all.js';
+
+const todoHTML = document.querySelector('.todo_list');
+const todoInput = document.querySelector('.todo_input');
+const clearAllBtn = document.querySelector('.clear_btn');
+
+const todoList = {
+  todoTasks: [],
+};
+
+const setLocalStore = (todoTasks) => {
+  localStorage.setItem('tasks', JSON.stringify(todoTasks));
+};
+
+const addTask = (desctiption, completed = false, index, todoList) => {
+  const task = {
+    desctiption,
+    completed,
+    index,
+  };
+  todoList.todoTasks.push(task);
+  setLocalStore(todoList.todoTasks);
+};
+
+const removeTask = (i, todoList) => {
+  todoList.todoTasks.splice(i, 1);
+  todoList.todoTasks.forEach((task) => {
+    task.index = i + 1;
+  });
+  setLocalStore(todoList.todoTasks);
+};
+
+const editTask = (input, listItem, i, todoList) => {
+  if (input.readOnly) {
+    input.readOnly = false;
+    listItem.style.backgroundColor = 'rgb(182, 166, 166)';
+  } else {
+    input.readOnly = true;
+    todoList.todoTasks[i].desctiption = input.value;
+    setLocalStore(todoList.todoTasks);
+  }
+};
+
+const isCompleted = (complete, input, i, todoList) => {
+  if (complete) {
+    todoList.todoTasks[i].completed = false;
+    input.style.textDecoration = 'none';
+    setLocalStore(todoList.todoTasks);
+  } else {
+    todoList.todoTasks[i].completed = true;
+    input.style.textDecoration = 'line-through';
+    setLocalStore(todoList.todoTasks);
+  }
+};
+
+function displayTasks() {
+  todoHTML.innerHTML = '';
+  todoList.todoTasks.innerHTML = '';
+  todoList.todoTasks.forEach((task, i) => {
+    task.index = i + 1;
+
+    const listItem = document.createElement('article');
+    listItem.classList.add('list_item');
+    const listContent = document.createElement('div');
+    listContent.classList.add('list_content');
+    const checkbox = document.createElement('input');
+    checkbox.classList.add('checkbox');
+    checkbox.setAttribute('type', 'checkbox');
+    checkbox.setAttribute('id', task.index);
+    checkbox.setAttribute('name', task.index);
+    const input = document.createElement('input');
+    input.classList.add('todo_task');
+    input.setAttribute('type', 'text');
+    input.setAttribute('id', task.index);
+    input.setAttribute('name', task.index);
+    input.setAttribute('value', task.desctiption);
```

Comment:
you are giving the same id for different element while it should be unique, you can concat a string with the index to get a dynamic and different id for each element `id="checkbox${task.index}"`

Reply:
i mean by element input and checkbox  both gets the same id for each iteration